scripts/integration-helpers/framework.ts

```diff
@@ -109,9 +146,9 @@ export class TriggerEndToEndTest {
   firestoreFromRtdb = false;
   rtdbFromRtdb = false;
   firestoreFromFirestore = false;
-  cliProcess?: CLIProcess;
 
-  constructor(public project: string, private readonly workdir: string, config: FrameworkOptions) {
+  constructor(project: string, workdir: string, config: FrameworkOptions) {
+    super(project, workdir, config);
```

Comment:
aren't lines 86-94 a duplicate of lines 152-158? I think the code is doing the same thing twice

Reply:
ah i see, thanks, done.